### PR TITLE
fix: validation for return quantity and reason

### PR DESCRIPTION
**Pull request by rbedeoan-plenty**
[AB#92286](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/92286)

### CODE REVIEW

**Comment by plenty-to-azure** [comment]
✅ Successfully linked to Azure Boards work item(s):
- [Product Backlog Item 92286](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/92286)

File: apps/web/components/OrderReturnForm/OrderReturnForm.vue [hunk 1]
```diff
@@ -64,23 +64,42 @@ const emit = defineEmits(['close']);
...
-
+const { send } = useNotification();
 fetchReturnReasons();
 
+const runtimeConfig = useRuntimeConfig();
 const confirmation = ref(false);
 const selectAllItems = ref(false);
 
-const canInitiate = computed(() =>
+const quantitySelected = computed(() =>
```
Comment (maxiroellplenty) [inline apps/web/components/OrderReturnForm/OrderReturnForm.vue]: can we move this code into the composable?

File: apps/web/components/OrderReturnForm/OrderReturnForm.vue [hunk 2]
```diff
@@ -64,23 +64,42 @@ const emit = defineEmits(['close']);
...
   Object.values(returnData?.value?.variationIds || {}).some((item) => item.quantity >= 1),
 );
 
+const hasQuantityAndNoReasonsSelected = computed(() =>
```
Comment (maxiroellplenty) [inline apps/web/components/OrderReturnForm/OrderReturnForm.vue]: this check could also be handled in the composable

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

File: apps/web/nuxt.config.ts [hunk 3]
```diff
@@ -131,6 +131,7 @@ export default defineNuxtConfig({
       turnstileSiteKey: process.env?.CLOUDFLARE_TURNSTILE_SITE_KEY ?? '',
       newsletterFromShowNames: process.env?.NEWSLETTER_FORM_SHOW_NAMES === '1' ?? false,
       useWebp: process.env?.USE_WEBP === '1' ?? false,
+      validateReturnReasons: process.env.VALIDATE_RETURN_REASONS === '1' ?? false,
```
Comment (maxiroellplenty) [inline apps/web/nuxt.config.ts:134]: Add VALIDATE_RETURN_REASONS value to the .env.example

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**